migration/2.1.3/SequencerInbox.sol

```diff
@@ -1,54 +1,27 @@
-// Copyright 2021-2022, Espresso Systems.
+// Copyright 2021-2022, Offchain Labs, Inc.
 // For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE
 // SPDX-License-Identifier: BUSL-1.1
 
 pragma solidity ^0.8.0;
 
-import {
-    AlreadyInit,
-    HadZeroInit,
-    BadPostUpgradeInit,
-    NotOrigin,
-    DataTooLarge,
-    DelayedBackwards,
-    DelayedTooFar,
-    ForceIncludeBlockTooSoon,
-    ForceIncludeTimeTooSoon,
-    IncorrectMessagePreimage,
-    NotBatchPoster,
-    BadSequencerNumber,
-    AlreadyValidDASKeyset,
-    NoSuchKeyset,
-    NotForked,
-    NotBatchPosterManager,
-    NotCodelessOrigin,
-    RollupNotChanged,
-    DataBlobsNotSupported,
-    InitParamZero,
-    MissingDataHashes,
-    NotOwner,
-    InvalidHeaderFlag,
-    NativeTokenMismatch,
-    BadMaxTimeVariation,
-    Deprecated,
-} from "../libraries/Error.sol";
-import "./IBridge.sol";
-import "./IInboxBase.sol";
-import "./ISequencerInbox.sol";
-import "../rollup/IRollupLogic.sol";
-import "./Messages.sol";
-import "../precompiles/ArbGasInfo.sol";
-import "../precompiles/ArbSys.sol";
-import "../libraries/CallerChecker.sol";
-import "../libraries/IReader4844.sol";
-
-import {L1MessageType_batchPostingReport} from "../libraries/MessageTypes.sol";
-import "../libraries/DelegateCallAware.sol";
-import {IGasRefunder} from "../libraries/IGasRefunder.sol";
-import {GasRefundEnabled} from "../libraries/GasRefundEnabled.sol";
-import "../libraries/ArbitrumChecker.sol";
-import {IERC20Bridge} from "./IERC20Bridge.sol";
-import {IEspressoTEEVerifier} from "../bridge/IEspressoTEEVerifier.sol";
+import { AlreadyInit, HadZeroInit, BadPostUpgradeInit, NotOrigin, DataTooLarge, DelayedBackwards, DelayedTooFar, ForceIncludeBlockTooSoon, ForceIncludeTimeTooSoon, IncorrectMessagePreimage, NotBatchPoster, BadSequencerNumber, AlreadyValidDASKeyset, NoSuchKeyset, NotForked, NotBatchPosterManager, NotCodelessOrigin, RollupNotChanged, DataBlobsNotSupported, InitParamZero, MissingDataHashes, NotOwner, InvalidHeaderFlag, NativeTokenMismatch, BadMaxTimeVariation, Deprecated } from '../libraries/Error.sol';
```

Comment:
I think we can expand this part for readibility!

Reply:
I dont understand what you mean here?

Reply:
Just the formatting that changed and was better before